src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -40,13 +52,77 @@ export const PosterPreview: React.FC<PosterPreviewProps> = ({
     [categories]
   );
 
+  const { worldUsers } = useWorldUsers();
+
+  const userPresenter = useMemo<JSX.Element[]>(() => {
+    return worldUsers
+      .filter((user) => user.partyName === authorName)
+      .map((user) => (
+        <UserAvatar key={`user-${user.id}`} user={user} showStatus />
+      ));
+  }, [worldUsers, authorName]);
+
+  const recentPosterUsers = useRecentLocationUsers(posterVenue.name);
+
+  const numUsers = recentPosterUsers.recentLocationUsers.length;
+
+  const renderMoreInfoUrl = useMemo(() => {
+    if (!moreInfoUrl) return;
+    return (
+      <a href={moreInfoUrl} target="_blank" rel="noreferrer">
```

Comment:
instead of explicit "rel" probably 
```suggestion
      <a href={moreInfoUrl} target="_blank"  {...externalUrlAdditionalProps}>
```
should be used (would need an import ` from "utils/url"`

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1534/commits/f17936a2d37b1c1e621b314cd5867c28016f6073